docs/attribution.md

```diff
@@ -76,28 +70,86 @@ More details can be found on the GMTED2010 [about](http://topotools.cr.usgs.gov/
 
-More details can be found on the ETOPO1 [about](https://www.ngdc.noaa.gov/mgg/global/global.html) page and [data sources](https://www.ngdc.noaa.gov/mgg/global/etopo1sources.html) page, and [description](https://www.ngdc.noaa.gov/docucomp/page?xml=NOAA/NESDIS/NGDC/MGG/DEM/iso/xml/316.xml&view=getDataView&header=none) page and [disclaimer](https://www.ngdc.noaa.gov/ngdcinfo/privacy.html#disclaimer) page.
+### data.gov.uk LIDAR Composite Digital Terrain Model
+
+[Released](https://data.gov.uk/dataset/lidar-composite-dtm-2m1) by [data.gov.uk](https://data.gov.uk/), licensed under [Open Government License version 3](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)
+
+Attribution statement:
+
+> © Environment Agency copyright and/or database right 2015. All rights reserved.
+
+### data.gv.at Digitales Geländemodell (DGM) Österreich
+
+[Released](https://www.data.gv.at/katalog/dataset/b5de6975-417b-4320-afdb-eb2a9e2a1dbf) by [data.gv.at](https://www.data.gv.at/), licensed under [Creative Commons Namensnennung 3.0 Österreich](https://creativecommons.org/licenses/by/3.0/at/deed.de)
+
+Attribution statement:
+
+> © offene Daten Österreichs – Digitales Geländemodell (DGM) Österreich.
+
+### data.kartverket.no Digital terrengmodell
+
+[Released](http://data.kartverket.no/download/content/digital-terrengmodell-10-m-utm-33) by Kartverket, [licensed](http://www.kartverket.no/data/lisens/) under [Creative Commons Attribution 4.0 International (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/deed.no).
+
+Attribution statement:
+
+> © Kartverket
+
+### Arctic Digital Elevation Model (ArcticDEM)
+
+[Released](http://nga.maps.arcgis.com/apps/MapSeries/index.html?appid=cf2fba21df7540fb981f8836f2a97e25) without limitation. "ArcticDEM data is an unlicensed product and may be used, distributed, and modified without permission".
+
+Requested citation:
+
+> DEM(s) were created from DigitalGlobe, Inc., imagery and funded under National Science Foundation awards 1043681, 1559691, and 1542736.
+
+### Digital Terrain Model over Europe (EU-DEM)
+
+[Released](https://www.eea.europa.eu/data-and-maps/data/eu-dem#tab-metadata) by [European Environmental Agency](https://www.eea.europa.eu/).
+
+Attribution statement:
+
+> Produced using Copernicus data and information funded by the European Union - EU-DEM layers.
+
+### Canadian Digital Elevation Model (CDEM)
+
+[Released](http://ftp.geogratis.gc.ca/pub/nrcan_rncan/elevation/cdem_mnec/doc/CDEM_product_specs.pdf) under the [Open Government Licence Agreement for Unrestricted Use of Digital Data](http://open.canada.ca/en/open-government-licence-canada).
+
+Attribution statement:
+
+> Contains information licensed under the Open Government Licence – Canada.
+
+### National Institute of Statistics and Geography (INEGI)
+
+[Released]() under [INEGI's "free use of information" license](http://en.www.inegi.org.mx/inegi/terminos.html).
```

Comment:
- Released is missing a link. Either add one or remove the link markdown.
- Released under **Mexico's**

Reply:
Fixed in 455a9c6.